styx-scheduler-service/src/test/java/com/spotify/styx/BackfillTriggerManagerTest.java

```diff
@@ -282,28 +280,53 @@ public void shouldTriggerAndUpdateBackfillsSynchronously() throws Exception {
   }
 
   @Test
-  public void shouldNotContinueTriggeringBackfillsIfTriggerFails() throws Exception {
```

Comment:
Whaaa did we have a test to assert _that_?!

Reply:
:) so it was by design.